wandb/sdk/internal/system/assets/network.py

```diff
@@ -68,6 +68,71 @@ def aggregate(self) -> dict:
         return {self.name: aggregate}
 
 
+class NetworkTrafficSent:
+    """Network traffic sent."""
+
+    name = "network.traffic_sent"
+    samples: "Deque[float]"
+    last_sample: float
+
+    def __init__(self) -> None:
+
+        self.network_sent = NetworkSent()
```

Comment:
I'd make it independent from NetworkSent(), just measure it here

Reply:
@dmitryduev  would this mean just calling `NetworkSent().sample()` and assigning that to a different variable name? I'm a bit confused as to how to get the initial value for upload speed without calling NetworkSent()